features/account/customer_account/customer_profile_validation.feature

```diff
@@ -0,0 +1,43 @@
+@customer_account
+Feature: Customer profile validation
+    In order to avoid making mistakes when changing my personal information
+    As a logged user
+    I want to be prevented from changing fields to incorrect values
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has customer "claire@underwood.com"
+        And there is a customer account "Francis Underwood" with email "francis@underwood.com" identified by "whitehouse"
```

Comment:
`And there is a customer "Francis Underwood" identified by an email "francis@underwood.com" and password "whitehouse"`